### ISSUE TITLE
This plugin don't seems to be doing anything for me !

### ISSUE THREAD

**Issue by anuragvohraec** [axetroy/vscode-deno#80]
### test
Installed this plugin and tried using it as per the instruction:
Add .vscode/setting.json file in project folder.
restarted vscode

**Describe the bug**
No intellsense is provided for std imports from deno.land.
Even if I do write it manually say :

`import * as fs from 'https://deno.land/std/fs/mod.ts';`

Then too it complain about the .ts extension.

**To Reproduce**

Install and follow installation steps.

**Screenshots**

![image](https://user-images.githubusercontent.com/53807480/75618636-383e0780-5b97-11ea-81d5-576ca6fb0b33.png)

**Versions**

vscode: 1.42.1
deno: 0.35
extension: 3.0.1

**Comment by xpyxel** [comment]
You have to `deno fetch <file>` for deno to cache the file for the plugin to be able to see the import properly, also you have to reload the plugin after fetching so it sees the newly cached file.

**Comment by anuragvohraec** [comment]
> You have to `deno fetch <file>` for deno to cache the file for the plugin to be able to see the import properly, also you have to reload the plugin after fetching so it sees the newly cached file.

Yup, working after following what you say.

**Comment by David-Else** [comment]
I am afraid I can't make it work using the new version, I followed all the steps listed in this thread  and reloaded VS Code. I also deleted the contents of `$HOME/.cache/deno/deps and /gen` in case of conflicts. After running `deno fetch` all the files returned again:
![Screenshot from 2020-03-01 12-35-38](https://user-images.githubusercontent.com/12832280/75625781-3dac4980-5bb9-11ea-849e-f2ab1871b95a.png)

See screen shot, and here is the repo if it helps https://github.com/David-Else/eslint-config-octopus
![Screenshot from 2020-03-01 12-30-20](https://user-images.githubusercontent.com/12832280/75625714-ab0baa80-5bb8-11ea-9e44-826ca0d12b91.png)

**Comment by axetroy** [comment]
@David-Else 
Seems to be because `tsconfig.json` exists in your project directory

Try to delete it and it will work

**Comment by David-Else** [comment]
That is weird! I hope it can be fixed. I use Deno and normal TypeScript mixed together in many projects, so this bug is really bad for me.

**Comment by David-Else** [comment]
I can't seem to use the VSCode ESlint extension without a `tsconfig.json`
```
Parsing error: File '/home/david/sites/TypeScript/eslint-config-octopus-GITHUB/tsconfig.json' not found. ...eslint [1 , 1]
```
Also `rollup` uses it. Not being able to have a `tsconfig.json` at this point is impossible unless there is some workaround I am not aware of. I think it used to work, but maybe I am wrong.

**Comment by axetroy** [comment]
@David-Else 
I hvae fix this bug in v3.0.5 (9ce2874230d4c66ea657f5d2de19c38eb8719df6)

Now your tsconfig.json will not disable the extension

Just wait for a minute for CI green and then it will release the new version automatically

**Comment by David-Else** [comment]
Excellent work! It is now fixed.
